Std/Data/BitVec/Basic.lean

```diff
@@ -77,15 +77,26 @@ protected def add (x y : BitVec n) : BitVec n :=
 /-- Subtraction for bit vectors. -/
 protected def sub (x y : BitVec n) : BitVec n :=
   { val := x.val - y.val }
+/-- Bit vector of size `n` where all bits are `1`s -/
+protected def allOnes (n : Nat) : BitVec n :=
+  BitVec.sub (BitVec.ofNat n 0) (BitVec.ofNat n 1)
```

Comment:
Isn't this just `-1`?

Reply:
Yes.

Reply:
...so why not write it that way? (I see we also need a `Neg` instance to make it work.)

Reply:
Yes, we would need the `Neg`. Happy with this change.